### ISSUE TITLE
Impulse formulation

### ISSUE THREAD

**Issue by jstuder3** [CMM-22/a5#3]
Hey, I'm trying to work on the fourth task, but I'm having a bit of trouble understanding the definition of K_a in the notes:

![image](https://user-images.githubusercontent.com/87820315/166447771-6378beef-f4c1-4714-8089-45f3681f8f8f.png)

What is the meaning of the subscript x in the marked term? Do we just take the x-component of the respective vectors? Or is it supposed to be a cross product and the last x is just a typo?

Kind regards

**Comment by eastskykang** [comment]
r<sub>ax</sub> is a skew symmetric matrix i.e. [r<sub>a</sub>]<sub>x</sub>.

Please see Conversion to matrix multiplication section of https://math.fandom.com/wiki/Cross_product

**Comment by jstuder3** [comment]
Hey, thanks a lot for your reply! That helped a lot!

However, I ran into another question: Should the term "1/m_a" from K_a above be understood as a 3x3-matrix where all entries are 1/m_a or should it be a 3x3-matrix where the on-diagonal entries are 1/m_a and everything else is zero? With the first version, I got very funky behaviour for infinite friction, but it seems to work just fine when it's a diagonal matrix.
